src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend.jes
+
+import java.net.URL
+
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.util.ConfigUtil._
+
+case class JesAttributes(applicationName: String,
+                         project: String,
+                         executionBucket: String,
+                         endpointUrl: URL,
+                         authMode: GcsAuthMode,
+                         docker: Option[DockerAuthInformation])
+object JesAttributes {
+
+  def apply(): JesAttributes = {
+    val jesConf = ConfigFactory.load.getConfig("backend").getConfig("jes")
+    val refConf: Config = ConfigFactory.parseResources("jes.conf")
+
+    jesConf.checkValidWrapped(refConf, "Jes")
+    jesConf.warnNotRecognized(refConf, "Jes", "dockerAccount", "dockerToken")
+
+    val applicationName = jesConf.getString("applicationName")
+    val project = jesConf.getString("project")
+    val executionBucket = jesConf.getString("baseExecutionBucket")
+    val endpointUrl = jesConf.getURL("endpointUrl")
+    val authMode = GcsAuthMode.fromString(jesConf.getString("authenticationMode"))
+    val docker = for {
+      account <- jesConf.getStringOption("dockerAccount")
+      token <- jesConf.getStringOption("dockerToken")
+    } yield DockerAuthInformation(account, token)
+
```

Comment:
Is checkValid really the right thing here though? If we're saying (at least I think we are) "ok, check this is all valid. But actually in these situations it varies, so double check that. And this might have blown up but it's ok" it seems like it's not the right tool. This ties in to the convo you're having with @mcovarr in another spot